kuma/wiki/jinja2/wiki/translate.html

```diff
@@ -10,7 +10,6 @@
 {% block bodyclass%}translate{% endblock %}
 
 {% block extrahead %}
-  {% stylesheet 'home' %}
 {% endblock %}
```

Comment:
Something broke when I removed the whole block - I didn't dig into  it too deeply.